tests/integration/medusa_minio_integration_test.go

```diff
@@ -0,0 +1,61 @@
+package integration
+
+import (
+	. "github.com/k8ssandra/k8ssandra/tests/integration/steps"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var (
+	medusaMinioTestTable    = "medusa_test"
+	medusaMinioTestKeyspace = "medusa"
+)
+
+var _ = Describe("Medusa on MinIO", func() {
```

Comment:
Wasn't the plan to ditch ginkgo?

Reply:
Based on prior discussions, I was also under the impression that we were going to just using the `testing` package probably along with https://github.com/stretchr/testify.

Reply:
@jsanda and @adejanovski  I propose an informal write-up for the prior discussions about our testing approach, so we are all on the same page.  I can see where `Ginkgo` from an integration standpoint may still be viable w/ a more direct approach for the unit tests.

Looking at this `medusa_minio_integration_test.go` file, it is extremely readable, easy-to-understand, and self-documenting.

Reply:
>I propose an informal write-up for the prior discussions about our testing approach

An excellent suggestion and I think that would be great in keeping with the spirit of open source. Maybe we can try to do it with GH Discussions?

Reply:
Decision pending here. Discussion taking place [here](https://github.com/k8ssandra/k8ssandra/discussions/540).

Reply:
We already decided against BDD, lets not decide multiple times the same thing. Seriously.

Reply:
Actually we did not. We decided against GoDog, but not everyone is on the same page about BDD vs `testing`.
I agree though it's time to build a consensus, which yet has to happen.

Reply:
The whole meeting was about BDD, including our guest appearance. We need a reason to use BDD or framework at all beyond testing.T and there simply isn't any (and nothing in this PR has convinced more than the opposite).

Reply:
I think one thing that would help me (and possibly others), if we could review a set of `k8s non-BDD` Go tests used for integration and e2e testing to understand what that fully looks like with all the complexities.  

My concern is that these frameworks & libs (Ginkgo, Terratest, Gomega, etc.) were built to ease the pain and understanding of tests at the integration and e2e testing levels.